### DISCUSSION TITLE
App Window border color

### DISCUSSION THREAD

**Discussion by Olexandrio** [Eugeny/tabby#4831] [Q&A] [answered]
Recently met Tabby. I like him. Trying to customize it for my job.
I wanted to define white border of app window for the Tabby's dark theme. I need this because the window has a color similar to my dark OS background.
In "_Settings> Appearance> Custom CSS_", I have styled the border for "_app-root_" tag. However, when the app is running the "_app-root_" border is overridden. Using DevTools I saw that it overridden in file "_theme.scss_". However, I not found this source in app folders.
Can I override the app window border and where is correct to do it?
// ver. Alpha 159

**Comment by Eugeny** [comment, marked as answer]
Yes - you can increase priority of your own rules with `!important`:

```css
app-root {
  border: 2px solid red !important;
}
```

**Comment by lawweiliang** [comment]
@Olexandrio 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
